### PR TITLE
Update validation when password is set but obfuscated

### PR DESCRIPTION
**Pull request by ChomickiM**

### CODE REVIEW

**Review by psantos1113 — APPROVED** [review]

File: channel-email/src/test/java/com/synopsys/integration/alert/channel/email/validator/EmailGlobalConfigurationValidatorTest.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 import com.synopsys.integration.alert.common.rest.model.ValidationResponseModel;
 import com.synopsys.integration.alert.service.email.model.EmailGlobalConfigModel;
 
-public class EmailGlobalConfigurationValidatorTest {
+class EmailGlobalConfigurationValidatorTest {
```
Comment (bamandel) [inline channel-email/src/test/java/com/synopsys/integration/alert/channel/email/validator/EmailGlobalConfigurationValidatorTest.java:16]: What happened here with the removal of public?

File: component/src/test/java/com/synopsys/integration/alert/component/settings/proxy/validator/SettingsProxyValidatorTest.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
 import com.synopsys.integration.alert.common.rest.model.SettingsProxyModel;
 import com.synopsys.integration.alert.common.rest.model.ValidationResponseModel;
 
-public class SettingsProxyValidatorTest {
+class SettingsProxyValidatorTest {
```
Comment (bamandel) [inline component/src/test/java/com/synopsys/integration/alert/component/settings/proxy/validator/SettingsProxyValidatorTest.java:15]: This also lost public for some reason

File: channel-email/src/test/java/com/synopsys/integration/alert/channel/email/validator/EmailGlobalConfigurationValidatorTest.java (hunk 1 above)
Comment (ChomickiM) [reply channel-email/src/test/java/com/synopsys/integration/alert/channel/email/validator/EmailGlobalConfigurationValidatorTest.java:16]: This is to remove some of the Sonar code smells, I spoke with Paulo a few weeks back and he suggested when returning to these old tests we should just remove them to cut down on the number of remaining smells.

Comment (ChomickiM) [reply channel-email/src/test/java/com/synopsys/integration/alert/channel/email/validator/EmailGlobalConfigurationValidatorTest.java:16]: As an FYI the rule description is: JUnit5 test classes and methods should have default package visibility

**Review by bamandel — APPROVED** [review]

File: channel-email/src/test/java/com/synopsys/integration/alert/channel/email/validator/EmailGlobalConfigurationValidatorTest.java (hunk 1 above)
Comment (bamandel) [reply channel-email/src/test/java/com/synopsys/integration/alert/channel/email/validator/EmailGlobalConfigurationValidatorTest.java:16]: Interesting, I did not know this. Will make the switch as well in the future

**Pull request merged**